bundle-workflow/src/test_workflow/integ_test_suite.py

```diff
@@ -5,28 +5,72 @@
 # compatible open source license.
 
 import os
+import subprocess
 
+from git.git_repository import GitRepository
 from paths.script_finder import ScriptFinder
-from paths.tree_walker import walk
-from system.execute import execute
+from test_workflow.local_test_cluster import LocalTestCluster
 
 
 class IntegTestSuite:
-    def __init__(self, name, repo, test_recorder):
-        self.name = name
-        self.repo = repo
-        self.test_recorder = test_recorder
```

Comment:
this component is not tested with the new IntegTestSuite, which is otherwise working. I will integrate this back outside of this PR. Just want to make sure it doesn't break the working integ test workflow.